metadata/src/main/resources/common/metadata/DelegationTokenRecord.json

```diff
@@ -22,8 +22,10 @@
   "fields": [
     { "name": "Owner", "type": "string", "versions": "0+",
       "about": "The delegation token owner." },
+    { "name": "Requester", "type": "string", "versions": "0+",
+      "about": "The delegation token requester." },
```

Comment:
Normally we append new stuff at the end, but this record has never been used before, so I'm okay with inserting in the middle in this case.

Reply:
Understood. This is also a case where the original creator of the fields missed this field from TokenInformation which is what ZK uses today.